wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -981,3 +981,8 @@ def workflow_status_with_date(workflow_state):
         return _("Sent to %(task_name)s %(started_at)s") % translation_context
 
     return _("%(status_display)s %(task_name)s %(started_at)s") % translation_context
+
+
+@register.inclusion_tag("wagtailadmin/shared/human_readable_date.html")
+def human_readable_date(date, description=None):
+    return {"date": date, "description": description}
```

Comment:
```suggestion
    return {"date": date, "description": description,}
```
Minor nitpick but if you add a trailing comma here - black will format this to multi lines and hence the code will be easier to update in the future with less diffs.